spec/factories/miq_group.rb

```diff
@@ -1,7 +1,8 @@
 FactoryGirl.define do
   factory :miq_group do
+    guid { MiqUUID.new_guid }
+
     sequence(:description) { |n| "Test Group #{seq_padded_for_sorting(n)}" }
-    sequence(:guid) { |n| UUIDTools::UUID.parse_int((0x87654321 << 96) + n) }
```

Comment:
Wow, thanks @matthewd.  I had no idea what that number `87654321 ` was.  I'm fine with the proposed change.